docs/api/connection.rst

```diff
@@ -339,10 +339,339 @@ Connection
             The *callback* to be invoked after closing the connection.
 
 
-    .. js:method:: wrap(conn: AwaitConnection): Connection
-        :staticmethod:
-
-        Convert an :js:class:`AwaitConnection` into :js:class:`Connection`.
-
 .. _BigInt:
     https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt
+
+
+.. _edgedb-js-api-pool:
+
+Pool
+==========
+
+.. js:function:: createPool(options)
+                 createPool(options, callback)
+
+    Create a connection pool to an EdgeDB server.
```

Comment:
A quick async/await complete example would be cool: how to create a pool, how to run a `SELECT 1` in it, how to close the pool.

Reply:
I added an example to `usage.rst` file. Should I add the same example to `connection.rst`?